src/domain/repositories/InstanceRepository.ts

```diff
@@ -1,6 +1,5 @@
 import { UserSearch } from "../../data/entities/SearchUser";
 import { InstalledApp } from "../entities/InstalledApp";
-import { NamedRef } from "../entities/Ref";
 
 export interface InstanceRepository {
     getBaseUrl(): string;
```

Comment:
I would also remove `uploadFile` even though is incorrectly referenced in the ImportExportClient and I would change the reference there from InstanceRepository to DocumentRepository at least